apis/apps/v1alpha1/type.go

```diff
@@ -651,3 +651,236 @@ type ConnectionCredential struct {
 	// +optional
 	AccountName string `json:"accountName,omitempty"`
 }
+
+// List of all the built-in variables defined by KB.
+// When rendering the environment variables of Pod or Action, and the templates of config or script,
+// these variables will be passed in by default, so you can use them directly without declaration.
+// TODO: resources.
+// --------------------------------------------------------------------------------
+// |           |           |                      |  available in   |             |
+// | object    | attribute | var                  | template | Env  | description |
+// --------------------------------------------------------------------------------
+// | Namespace |           | KB_NAMESPACE         |          |      |             |
+// | Cluster   | name      | KB_CLUSTER_NAME      |          |      |             |
+// |           | UID       | KB_CLUSTER_UID       |          |      |             |
+// |           | comp name | KB_CLUSTER_COMP_NAME |          |      |             |
+// | Component | name      | KB_COMP_NAME         |          |      |             |
+// |           | replicas  | KB_COMP_REPLICAS     |          |      |             |
```

Comment:
Here, is there a need to further distinguish between static and dynamic variables? Variables like KB_COMP_REPLICAS are dynamic, and the behavior of horizontal scaling will not be reflected in real time to KB_COMP_REPLICAS. If the configuration rendering or env reference behavior depends on the dynamic KB_COMP_REPLICAS, it needs to be used with caution. The same goes for other dynamic variables.

Reply:
done